python/tflite_micro/signal/tflm_signal.bzl

```diff
@@ -1,5 +1,7 @@
 """Build rule for wrapping a custom TF OP from .cc to python."""
 
+load("@rules_python//python:defs.bzl", "py_library")
```

Comment:
Do we also need to remove the `native.` from the `py_library` usage down at line 64?

Reply:
thanks. fixed.